packs/mssql/actions/execute.query_and_email.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: "execute.query_and_email"
+description: "Executes an MS SQL Query and emails results as attachments"
+enabled: true
+
+runner_type: "action-chain"
+entry_point: "workflows/execute.query_and_email.yaml"
+
+parameters:
+  query_string:
```

Comment:
It looks like this action (unlike others) doesn't support parameter expansion and you need to provide all the parameters in-line in the `query_string` parameter.

Is this correct?